src/node/node_cfg_extra.ml

```diff
@@ -0,0 +1,109 @@
+(*
+Copyright (2010-2014) INCUBAID BVBA
```

Comment:
(several files) you should have a "Copyright 2016 iNuron NV" header here. (look at arakoon_etcd.ml)